src/api/auth/auth.controller.ts

```diff
@@ -10,12 +10,19 @@ export class AuthController {
   constructor(private readonly authService: AuthService) {}
 
   @Post('sign-up')
-  async register(@Body() register: CreateUserDto) {
+  async register(@Body() register: CreateUserDto, hre: any) {
+    let var1;
+    if (register.username === '') {
+      var1 = 'Username is required';
+    } else {
+      var1 = '';
+    }
```

Comment:
🚫 **[eslint]** <[unicorn/prefer-ternary](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v56.0.0/docs/rules/prefer-ternary.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This `if` statement can be replaced by a ternary expression.
```suggestion
    var1 = register.username === '' ? 'Username is required' : '';
```

<!-- __reviewdog__:ChA2NTM4ODAwZWE3YTMzOTBjEgZlc2xpbnQ= -->